src/components/contributions/contributions.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import EditIcon from './edit-icon.svg'
+
+const Contributions = ({ data }) => {
+  const commit = data.github.repository.object.history.nodes[0]
+  const file = data.github.repository.object.file
+  const lastUpdated = new Date(commit.author.date).toLocaleString('en-AU', {
+    dateStyle: 'long',
+  })
+  const editLink = `https://github.com/NUbots/NUbook/edit/main/${file.path}`
+  return (
+    <div className='text-sm text-secondary dark:text-secondary-inverted mb-6 border-t-2 border-b-2 border-gray-200 dark:border-gray-800 py-3 flex'>
+      <div className='flex-grow'></div>
+      <div className='pr-3 capitalize'>
+        <a
+          href={commit.url}
+          target='_blank'
+          className='lg:normal-case'
+          rel='noreferrer'
+        >
+          <span className='hidden lg:inline'>Last</span> updated{' '}
```

Comment:
Any reason why `updated` is outside of the span?

Reply:
This was done so that on mobile 'Last' is hidden and only 'Updated DATE' is shown.

Reply:
Gotcha. Nice!